config/locales/sufia.en.yml

```diff
@@ -166,3 +168,5 @@ en:
       department:
         text: "Department"
         label_html: "Department"
+      open:
+         label_html: Open Access
```

Comment:
Default label is "Public" I'm changing that here. Is this not the right place to override the label?